README.md

```diff
@@ -1,31 +1,95 @@
+[Orion](https://github.com/telefonicaid/fiware-orion).
 
--   [Architecture](documentation/architecture.md)
--   [Logs & Alarms](documentation/logs.md)
--   O&M
+This project is part of [FIWARE](https://www.fiware.org). You can find more FIWARE components in the
+[FIWARE catalogue](https://catalogue.fiware.org).
 
-    -   [Deployment](documentation/deployment.md)
-    -   [Configuration](documentation/configuration.md)
-    -   [Administration](documentation/admin.md)
-    -   [Metrics API](documentation/metrics_api.md)
+## Content
 
--   [API](documentation/api.md)
-    -   [Plain rules](documentation/plain_rules.md)
-    -   [Errors](documentation/errors.md)
--   [Data model](documentation/models.md)
--   [Available actions for PEP](documentation/pep_actions.md)
--   [Documentation for developers of perseo](documentation/development.md)
+-   [Background](#background)
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [API](#api)
+-   [Testing](#testing)
+-   [Quality Assurance](#quality-assurance)
+-   [More Information](#more-information)
+-   [License](#license)
+
+## Background
+
+Perseo is an [Esper](http://www.espertech.com/esper/)-based Complex Event Processing (CEP) software supporting the
+NGSIv2 protocol.
+
+It follows a straightforward idea: listening to events coming from context information to identify patterns described by
+rules, in order to immediately react upon them by triggering actions.
+
+![Perseo Components](docs/images/PerseoComponents.png)
+
+By leveraging on the
+[notifications mechanism](http://fiware-orion.readthedocs.io/en/latest/user/walkthrough_apiv2/index.html#subscriptions),
+clients instruct Orion CB to notify Perseo of the changes in the entities they care about (`Event API`). Details of this
+process are explained in the [Orion Subscription part of the User Manual](user/index.md#orion-subscription). Then, rules
+to the CORE Rule Engine can be easily managed using the publicly available WireCloud operational dashboard, or making
+use of any of the REST clients able to programmaticly use the Perseo's `Rule API`. These rules will identify patterns
+that will trigger actions with Orion to create or update entities, or with other different components or external
+systems, such as Web (HTTP), Email (SMTP) or SMS (SMPP) servers.
+
+## Installation
+
+The instructions to install Perseo can be found in the
+[Installation Guide](https://perseo.readthedocs.io/en/latest/setup.html)
+
+## Usage
+
+Information about how to use Perseo can be found in the
+[User & Programmers Manual](https://perseo.readthedocs.io/en/latest/example1.html)
+
+## API
+
+APIs and examples of their usage can be found [here](https://perseo.readthedocs.io/en/latest/api.html)
+
+## Testing
+
+For performing a basic end-to-end test, you can follow the detailed instructions
+[here](https://perseo.readthedocs.io/en/latest/test.html).
+
+## Quality Assurance
```

Comment:
This section is not currently showing meaningful information, please remove it.

In addition, I have expressed my concerns to FIWARE staff about the current approach for QA sections (I'd send it to you by email).

Reply:
Ok. Fixed in f4ddd63de99c5a2748d72789121823cceb33f222